app/config/processing.js

```diff
@@ -5,6 +5,7 @@ const schema = Joi.object({
   scheduleProcessingMaxElapsedTime: Joi.number().default(300000), // 5 minutes
   scheduleProcessingMaxBatchSize: Joi.number().default(100),
   constructionActive: Joi.boolean().default(true),
```

Comment:
Yes it should.  I thought that too and the only reason I didn't rename it was because the config values in the pipeline are already set to use `constructionActive`.  But that's no excuse to add tech debt in the form of confusion!

I've created a sub task to update the config store and have renamed this as suggested.